prisma/schema.prisma

```diff
@@ -15,9 +15,9 @@ model Log {
   id        String   @id @default(cuid())
   method    String
   url       String
-  headers   String
+  headers   Json
   body      Json
-  response  Json
+  response  Json?
```

Comment:
Thanks for this great work, I needed to make the `response` nullable with a new migration, otherwise I was getting errors when testing it locally